Libs/MRML/Core/vtkMRMLSegmentationDisplayNode.h

```diff
@@ -58,6 +58,8 @@ class VTK_MRML_EXPORT vtkMRMLSegmentationDisplayNode : public vtkMRMLDisplayNode
     double Opacity3D{1.0};
     double Opacity2DFill{1.0}; // This one is used for labelmap volume related operations (color table, merged labelmap)
     double Opacity2DOutline{1.0};
+    /// Pickable by interactions
```

Comment:
This is not very clear. I've kind of almost already forgotten what exactly we wanted to use this for and how it is related to the `Selectable` node property, and this short comment did not help me to understand what this flag is for. Maybe also mention that it is only for 3D views.